src/services/superfluidMinionService.js

```diff
@@ -0,0 +1,142 @@
+import Web3 from 'web3';
+
+import SuperfluidMinionAbi from '../contracts/superfluidMinion.json';
+import { SUPERFLUID_MINION_STREAMS } from '../graphQL/superfluid-queries';
+import { TokenService } from './tokenService';
+import { chainByID, getGraphEndpoint } from '../utils/chain';
+import { graphFetchAll } from '../utils/theGraph';
+
+import SuperfluidSDK from '@superfluid-finance/js-sdk';
+
+let sf = null;
+
+const getSuperTokenBalances = async (chainID, minion, superTokens) => {
+  const tokenBalances = superTokens.map(async (tokenAddress) => {
```

Comment:
try catch

Reply:
this will be reviewed again in the next PR update